### DISCUSSION TITLE
Six row Shulker Boxes.

### DISCUSSION THREAD

**Discussion by VonSparq** [PurpurMC/Purpur#915] [__Dismissed__] [answered]
Would it be possible for you to please add an option to make Shulker Boxes have six row inventory just like there is an option like that for Ender Chests and Barrels?

**Comment by Roan-V** [comment, marked as answer]
No it seems, sadly
https://discord.com/channels/685683385313919172/685683802479001625/865815419138080788
Billy was talking about bundles
> I cant change anything about them server side 😦
client completely ignored all the changes I made
tried making them hold 128 items, including shulker boxes. no-go on both

https://discord.com/channels/685683385313919172/685683802479001625/782995422696177665
> messing with shulker boxes really starts to screw things up quickly. i've dabbled with that idea before, and it was extremely buggy (including duped items and shulker-ception). i've also tried making them have 6 rows, but that led to a lot of duped/missing items inside. i even tried making them open on right click from inventory, but that creates the same shulker-ception bug for some reason. i've pretty much given up on shulkers at this point lol

**Reply by VonSparq** [reply]
Thank you very much for clarifying, it is very sad that this can't be done as it would have been an amazing tweak!
